WordPressKit.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressKit'
-  s.version       = '14.1.0'
+  s.version       = '15.0.0-beta.1'
```

Comment:
Usually we make version change in a dedicated PR along side with moving changelog content. Do you intend to make a release once this PR is merged?

Reply:
Oh, right. I just wanted to put a line in the sand for the breaking change.